keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -729,6 +736,8 @@ that might indicate a serious problem?
 
 #### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
+We have unit tests testing the "rollback" scerario as we do test with positive and negative cases of the feature.
+
 <!--
 Describe manual testing that was done and the outcomes.
```

Comment:
You should also perform the manual testing that actually creates a cluster, upgrades and downgrades.

Reply:
Did a manual test following the path that @mimowo did in another KEP.